src/controllers/routers/referees.js

```diff
@@ -61,18 +53,8 @@ const getAll = async (req, res) => {
 const getReferee = async (req, res, next) => {
```

Comment:
1. change function name to getById
2. test sending an alphanumeric ID; if ID is not. integer then we should send a RESOURCE NOT FOUND error

Reply:
changed function name to getById
update error handling to test alphanumeric ID

Reply:
please test GetById; does not work even for valid Ids

Reply:
updated getById by including isValidId to check if id can be parsed into an int